woocommerce/google-listings-and-ads
# Add Rebranding Tour

### Changes proposed in this Pull Request:

PT: pcTzPl-2i9-p2

This PR creates a tour component that shows when you enter the "Google for WooCommerce" pages.

Design approval for the text: pcUrMV-7Wk-p2#comment-22473

### Screenshots:

<img width="1708" alt="Screenshot 2024-06-28 at 16 05 27" src="https://github.com/woocommerce/google-listings-and-ads/assets/5908855/3c4060bf-fbfa-47fd-90cb-188454b9c616">

### Detailed test instructions:


1. Checkout this PR and enter in any Google for WooCommerce page 
2. See the tour as in the screenshot
3. Close the tour
4. Refresh page
5. Tour doesn't show


### Additional details:


### Changelog entry

>
